department-of-veterans-affairs/caseflow
# Ryan/appeals 24482

Resolves [#{24482}](https://jira.devops.va.gov/browse/APPEALS-24482)

# Description
Added rake task that generates legacy appeals on the first 12 veterans. Also modified Case.rb generator to work with it.

## Acceptance Criteria

## Testing Plan
1. Go to #{jira issue link} or list them below
in a terminal, run `bundle exec rake db:generate_legacy_appeals`
2. Check any Veteran who has an id of 1-12 in your database inside of caseflow queue.
3. Validate that the veteran now has several legacy appeals that were generated.
